ParcialProgramacion4/src/Empleado.java

```diff
@@ -60,4 +60,60 @@ public void mostrarHabilidades() {
         }
     }
 
+
+    public boolean puedeAplicar(Hashtable<Habilidad,Integer> requisitos) {
+        //si esta en un puesto jerarquico, determino si cumple con annos minimos en ese puesto
+        Cargo cargoActual = this.getCargoActual();
+        boolean jerarquicoCumpleAnnosMinimos = cargoActual.jerarquicoCumpleAnnosMinimos();
+
+        //comparo requisitos con habilidades
+        boolean cumpleRequisitos = this.cumpleRequisitos(requisitos);
+
+        return jerarquicoCumpleAnnosMinimos && cumpleRequisitos;
+    }
```

Comment:
Quizás ya es mucha sinterización, pero el return podría devolver:

return (cargoActual.jerarquicoCumpleAnnosMinimos()) && (this.cumpleRequisitos(requisitos));